rfc6962-bis.xml

```diff
@@ -1470,12 +1470,12 @@ but it is expected there will be a variety.
         </section>
       </section>
       <section title="TLS Feature Extension">
-	<t>
-	  If a certificate in a chain includes the <xref target="tls_transinfo_extension">transparency_info</xref> extension number in a <xref target="RFC7633">TLS Feature Extension</xref>, then CT compliance is required for all certificates which are directly or indirectly issued by that certificate.
-	</t>
-	<t>
-	  Clients MUST treat certificates which fail this requirement as non-compliant.
-	</t>
+        <t>
+          If any certificate in a chain includes the <xref target="tls_transinfo_extension">transparency_info</xref> TLS extension identifier in the <xref target="RFC7633">TLS Feature</xref> certificate extension, then CT compliance (using any of the mechanisms from <xref target="tls_servers"/>) is required.
+        </t>
+        <t>
+          Clients MUST treat certificates which fail this requirement as non-compliant.
```

Comment:
I think we discussed "Clients MUST" yesterday.  You pointed out that it's not just TLS clients that evaluate certificate chains.  However...
How would any other type of client evaluate whether or not CT compliance has been achieved?
It's only a TLS client that will receive SCTs / inclusion proofs that are sent via the transparency_info TLS extension, for example.